dns/ddclient/src/opnsense/service/templates/OPNsense/ddclient/ddclient.conf

```
@@ -47,6 +47,7 @@ protocol=1984, \
 {%      elif account.service == 'godaddy' %}
```

```suggestion
{%      elif account.service in ['godaddy', 'gandi'] %}
```

Can you fold these to cases into one? remove gandi block below